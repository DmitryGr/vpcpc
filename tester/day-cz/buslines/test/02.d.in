90
64 35
27 38
53 38
41 76
9 50
21 45
71 33
31 82
69 16
43 36
24 63
81 80
38 61
89 10
41 15
68 2
5 38
89 38
69 36
87 28
78 73
38 48
78 50
47 38
2 10
88 57
62 61
45 13
23 89
24 9
9 37
38 66
38 67
59 22
38 51
29 49
83 38
70 82
84 64
75 89
28 12
11 24
38 54
38 7
13 35
29 8
71 69
4 7
26 38
46 78
15 63
42 34
64 44
88 38
33 18
39 38
38 72
82 12
38 80
29 77
26 20
83 25
30 32
55 48
38 29
22 38
3 87
56 43
84 46
19 1
56 70
74 38
32 38
60 38
58 46
13 18
14 38
19 38
85 38
79 38
86 38
38 90
2 76
17 38
18 40
6 38
24 52
90 34
65 38
