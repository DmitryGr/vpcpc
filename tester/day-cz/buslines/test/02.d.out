89
2064
89
89
89
89
177
89
1857
2039
89
347
1399
89
1959
89
89
1213
177
89
89
177
89
1974
89
177
89
263
350
89
89
177
1077
177
1419
809
89
3083
89
89
1979
89
737
89
177
1672
89
177
89
1763
89
89
89
89
89
663
89
89
89
89
177
89
1937
1537
89
89
89
89
957
587
1013
89
89
89
89
1997
89
1752
89
177
89
513
177
1563
89
89
177
177
2144
263
